oshi-core/src/main/java/oshi/hardware/platform/linux/LinuxSensors.java

```diff
@@ -41,30 +41,57 @@
 
     // Base HWMON path, adds 0, 1, etc. to end for various sensors
     private static final String HWMON = "/sys/class/hwmon/hwmon";
+    // Base THERMAL_ZONE path, adds 0, 1, etc. to end for temperature sensors
+    private static final String THERMAL_ZONE = "/sys/class/thermal/thermal_zone";
 
     // Map from sensor to path
     private Map<String, String> hwmonMap = new HashMap<>();
```

Comment:
No objections.  Trying to remember why I added `this` everywhere... I think some automated code review tool recommended it as a defensive practice.